index.d.ts

```diff
@@ -1,24 +0,0 @@
-export interface SimpleIcon {
```

Comment:
This file should not be renamed to `types.d.ts` instead of be removed?

Reply:
good catch, fixed that